dotCMS/src/main/java/com/dotmarketing/db/DbConnectionFactory.java

```diff
@@ -35,6 +36,40 @@
     protected static final String MSSQL = "Microsoft SQL Server";
     protected static final String H2 = "H2";
 
+    /**
+     * Gets the autoCommit for the current connection
+     * @return boolean
+     */
+    public static boolean getAutoCommit() {
+
+        boolean autoCommit = false;
+
+        try {
+            autoCommit = getConnection().getAutoCommit();
+        } catch (SQLException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)